packages/core/components/DataTable/DataTable.tsx

```diff
@@ -125,6 +125,16 @@ const DataTable = (props: DataTableProps) => {
     maxHeight: config.table.limitHeight && `${config.table.height}px`,
     OverflowY: 'scroll'
   }
+  const [height, getTableHeight] = useState(0)
```

Comment:
These three react hooks shouldn't be called conditionally. Use a more descriptive variable here, `height` could refer to a number of different heights throughout the app. Change `getTableHeight` to `setTableHeight` to follow React conventions.

Reply:
ok will reach in Teams